lib/vbox/vbox.go

```diff
@@ -5,14 +5,22 @@ import (
 	"encoding/json"
 	"fmt"
 	"os"
-	"sync"
 
-	log "github.com/Sirupsen/logrus"
 	virtualbox "github.com/riobard/go-virtualbox"
-	"github.com/xshellinc/iotit/lib"
-	"github.com/xshellinc/tools/constants"
 	"github.com/xshellinc/tools/dialogs"
 	"github.com/xshellinc/tools/lib/help"
+	"github.com/xshellinc/tools/lib/ssh_helper"
+)
+
+const (
+	VBoxName = "iotit-box"
```

Comment:
__[golint]__
* [golint] exported const VBoxName should have comment (or a comment on this block) or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->